docs/backup-home.md

```diff
@@ -88,7 +88,7 @@ backup:
 
 ## Remote server over SSH
 
-This location scheme is using borg on a remote server, with an SSH connection. Therefore, the borg software need to be
+This location scheme is using borg on a remote server, with a SSH connection. Therefore, the borg software needs to be
```

Comment:
Nope, sorry. We say an SSH, not a SSH ;-) The rule is how to pronounce the next word, not the console.

Reply:
All the rest is good! Thanks a lot for the amending, very useful to developers and users too.

Reply:
This is fine, I merge and will amend this ;-)

Reply:
> Nope, sorry. We say an SSH, not a SSH ;-) The rule is how to pronounce the next word, not the console.

Ha, the sound… ok, noted, thanks :)